### PR TITLE
Fix /"

### LINKED ISSUE #1301: Typing /" bypasses the chat handler
This was mentioned by OFF/tlworks before but was never reported here. Players are able to bypass mutes etc by typing /" in chat. This breaks the whole handler and chat is no longer handled by our hook. If I remember correctly the issue lies in the `ParseParameters(string)` method. I believe the nature of that code is what causes this (keep in mind that I am not 100% sure about this).

### PR DESCRIPTION
**Pull request by Simon311**
Fixes #1301.

### CODE REVIEW

File: TShockAPI/Commands.cs [hunk 1]
```diff
@@ -628,12 +628,29 @@ public static bool HandleCommand(TSPlayer player, string text)
 			if (cmdPrefix == SilentSpecifier)
 				silent = true;
 
-			var args = ParseParameters(cmdText);
-			if (args.Count < 1)
-				return false;
+			int index = -1;
+			for (int i = 0; i < cmdText.Length; i++)
+				if (IsWhiteSpace(cmdText[i]))
```
Comment (nicatronTg) [inline TShockAPI/Commands.cs]: Whether or not this is syntactically correct or not, stylistically there should be some brackets around this for loop.

**Comment by nicatronTg** [comment]
Let me clarify that last comment: I tested and made sure that command parsing wasn't changed too.

**Comment by Simon311** [comment]
Let me fix something quickly

**Comment by Simon311** [comment]
Once this is accepted, I will push commits and make another PR fixing #1206.

**Pull request merged**